src/Uno.UI/UI/Xaml/Media/RectangleGeometry.cs

```diff
@@ -33,6 +33,8 @@ public Rect Rect
 
 		#endregion
 
+		internal UIElement ClipOwner { get; set; }
```

Comment:
Same here : I would assume that we are able to re-use a `Geometry` on multiple elements, we should not introduce the concept of `[Clip]Owner`.